scripts/test-sim-backwards.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -ueo pipefail
+set +x
+
+TMP=$(mktemp -d -t tmp.XXXXXXXXXX)
+export STORJ_NETWORK_DIR=$TMP
+cleanup(){
+    rm -rf "$STORJ_NETWORK_DIR"
+    echo "cleaned up test successfully"
+}
+trap cleanup EXIT
+
+BRANCH_DIR="$STORJ_NETWORK_DIR/branch"
```

Comment:
shouldnt you be using `git describe --tags `git rev-list --tags --max-count=2`` this command? to get the current (latest) and the previous release?

Reply:
no, we should be testing the latest release and the current branch (or master branch in the future)